docs/shared/core-features/automate-updating-dependencies.md

```diff
@@ -66,6 +66,28 @@ After you run all the migrations, you can remove `migrations.json` and commit an
 
 Note: You may want to keep the `migrations.json` until every branch that was created before the migration has been merged. Leaving the `migrations.json` in place allows devs to run `nx migrate --run-migrations` to apply the same migration process to their newly merged code as well.
 
+## Keep Nx Packages on the Same Version
+
+When you run `nx migrate`, the `nx` package and all the `@nx/` packages get updated to the same version. It is important to keep these versions in sync, or you can encounter some difficult to debug errors. As long as you run `nx migrate` instead of manually changing the version numbers, you shouldn't have to worry about it.
+
+{% callout type="note" title="Use the latest nx-cloud version" %}
+The `nx-cloud` package does not need to be in sync with the other Nx packages. For best results, stay on the latest version of `nx-cloud`. The latest `nx-cloud` version supports the most recent 2 major versions of `nx`, although earlier versions of `nx` may also be compatible.
+{% /callout %}
+
+## Migrate Community Plugins Individually
+
+The `nx migrate` command migrates all the official plugins, but not the community plugins you have installed. Those plugins should be migrated individually with:
```

Comment:
I think this should be rephrased.

To me, this sounds like `nx migrate` *should* migrate community plugins but does not. I think it should be more positive in its wording to say `nx migrate` also works for community plugins and should be used to update those as well.